anyway/views/news_flash/api.py

```diff
@@ -257,14 +255,14 @@ def filter_by_resolutions(query, resolutions: List[str]):
         ands.append(
             and_(
                 NewsFlash.resolution == BE_CONST.ResolutionCategories.SUBURBAN_ROAD.value,
-                NewsFlash.road_segment_name != None,
+                NewsFlash.road_segment_name is not None,
             )
         )
     if "street" in resolutions:
         ands.append(
             and_(
                 NewsFlash.resolution == BE_CONST.ResolutionCategories.STREET.value,
-                NewsFlash.street1_hebrew != None,
+                NewsFlash.street1_hebrew is not None,
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969